packages/modelviewer.dev/index.html

```diff
@@ -506,6 +506,15 @@ <h3 class="grouping-title">Methods</h3>
               necessary. This is currently effectively the same as interacting
               with the poster via user input.</p>
             </li>
+            <li>
+              <div>exportScene(options)</div>
+              <p>Exports the model in glTF or GLB formats. Returns a promise
+              that resolves in a Blob of type "application/octet-stream" or
+              "application/json" depending on the export format.
+              For a comprehensive list of parameters see <a href='https://threejs.org/docs/index.html#examples/en/exporters/GLTFExporter.parse'>
+              GLTF exporter</a> documentation
```

Comment:
We should mention here what we've set animations and truncateDrawRange to, since we're only exposing a subset of the options.